packages/components/src/popover/index.js

```diff
@@ -73,50 +75,49 @@ const ArrowTriangle = ( props ) => (
 	</SVG>
 );
 
-const slotNameContext = createContext();
-
-const positionToPlacement = ( position ) => {
-	const [ x, y, z ] = position.split( ' ' );
+const MaybeAnimatedWrapper = forwardRef(
+	(
+		{
+			style: receivedInlineStyles,
+			placement,
+			shouldAnimate = false,
+			...props
+		},
+		forwardedRef
+	) => {
+		// In your component
```

Comment:
Might be good to update/remove this comment before merging.

Reply:
✅